<launch>

    <node pkg="global_planning" name="test_opencv" type="test_opencv" output="screen" /> 

</launch>
